pyro/ops/gaussian.py

```diff
@@ -207,6 +207,49 @@ def event_logsumexp(self):
                 chol_P.diagonal(dim1=-2, dim2=-1).log().sum(-1))
 
 
+class AffineNormal:
```

Comment:
nit: do we need to subclass `object` here?

Reply:
Python 3 no longer requires `(object)`. I'll also remove `(object)` from `Gaussian` in this file, but let's do general cleanup in a separate PR.